plugins/auth-backend-module-oidc-proxy-provider/config.d.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  auth?: {
+    providers?: {
+      /**
+       * Configuration for the Holos Platform identity aware proxy auth provider.
+       */
+      oidcProxy?: {
+        [authEnv: string]: {
```

Comment:
This entire layer is usually not present on proxy providers. The `createProxyAuthenticator` function will try to locate config that does not have this extra nesting.

Reply:
It's a reflection of [gcpIap?: {](https://github.com/jeffmccune/backstage/blob/305475c1b374cae3302fdfdad97c1be296e13af5/plugins/auth-backend-module-gcp-iap-provider/config.d.ts#L23-L24)

Let me know if you want something different, otherwise leaving this as-is.